### PR TITLE
Simply HTTPS Proxy Example

### PR DESCRIPTION
**Pull request by tobya**

### CODE REVIEW

File: httpsproxy/caddyfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+mysite.example.com {
+    
+    proxy / mysite.example.com:3333
+}
```
Comment (mholt) [inline httpsproxy/caddyfile]: This could simply become:

```
mysite.example.com
proxy / mysite.example.com:3333
```

File: httpsproxy/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+HTTPS an existing website
```
Comment (mholt) [inline httpsproxy/readme.md]: "Add HTTPS to an existing website" maybe

File: httpsproxy/readme.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+------------------------
+
+Setting up a very simple proxy to serve your existing site over **HTTPS** without changing your existing installation *(other than the port its served on)*.
+
+The caddyfile shows how you can use caddy to listen on 80 & 443 and sit infront of another webserver such as apache which is serving on differnt port (3333 in caddyfile) .
```
Comment (mholt) [inline httpsproxy/readme.md]: "Caddyfile", "sit in front", "Apache", "(3333 in this case; see Caddyfile)."

File: httpsproxy/readme.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+Instant HTTPS for an existing website without rewrite.
```
Comment (mholt) [inline httpsproxy/readme.md]: I'm not sure what this sentence means.

File: httpsproxy/caddyfile (hunk 1 above)
Comment (tobya) [inline httpsproxy/caddyfile]: I wasnt sure if that syntax could be used for more than 1 site, as I have multiple sites (each with {} ) in my own caddy file.  

I will update the file in the example though.

Comment (mholt) [inline httpsproxy/caddyfile]: Multiple sites need the braces, but your example only has one site.

**Comment by mholt** [comment]
@tobya  Did you mean to close this? It's looking good now.

**Comment by mholt** [comment]
Nevermind, I see the new one you made.

**Pull request closed without merging**